wg-structured-logging/charter.md

```diff
@@ -0,0 +1,69 @@
+# WG Structured Logging Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README]
+and uses the Roles and Organization Management outlined in [sig-governance].
+
+[sig-governance]: https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md
+[Kubernetes Charter README]: https://github.com/kubernetes/community/blob/master/committee-steering/governance/README.md
+
+## Scope
+
+Modernize logging in Kubernetes core components, allowing users to efficiently consume, process, store and analyse 
+information stored in logs.
+
+### In Scope
+
+- Define the standard for logging - propose libraries, interfaces, metadata schema
+- Reduce friction for using logging - reduce dependencies and performance overhead
+- Give more choice over logging - allow pluggable logging implementation
+- Ensure quality consistent logging - overview migration, create documentation, tooling and educate reviewers
+- Prevent regressions caused by logging - measure performance overhead and log volume changes
+
+For all the above points, we will focus on core Kubernetes components and addons.
+Other SIG subprojects/components (e.g. SIG Scheduling descheduler) are out of
+scope. However, we expect that tooling created by the group should be easy to adapt more broadly.
+
+### Out of scope
+
+- Logging of non-core Kubernetes components
+
+## Stakeholders
+
+Stakeholders in this working group span multiple SIGs that own parts of 
+the code in core Kubernetes components and addons.
+
+  - API Machinery
+  - Architecture
+  - Cloud Provider
+  - Instrumentation
+  - Network
+  - Node
+  - Scheduling
+  - Storage
+
+## Deliverables
+
+The artifacts the group is supposed to deliver include:
+- Completion of [Structured Logging migration]
+- Graduation of JSON logging format to GA
+- Documented guidelines on using Structured Logging
+- Automated tooling to prevent regressions caused by logging
+
+[Structured Logging migration]: https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1602-structured-logging
+
+## Roles and Organization Management
+
+This sig adheres to the Roles and Organization Management outlined in
```

Comment:
sig or wg? do other wg's link to sig-governance or wg-governance?

Reply:
Done. replaced with wg-governance and fixed the link